.github/workflows/backup.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: Create external backup
+
+# yamllint disable-line rule:truthy
+on:
+  schedule:
+    # Run every day at 10am
+    - cron: '0 10 * * *'
+  workflow_dispatch:
+
+jobs:
+  create-backup:
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: backup
+    env:
+      RCLONE_HOST: ${{ secrets.RCLONE_HOST }}
+      RCLONE_USER: ${{ secrets.RCLONE_USER }}
+      RCLONE_PASS: ${{ secrets.RCLONE_PASS }}
+      BORG_SSHKEY: ${{ secrets.BORG_SSHKEY }}
+      BORG_REPO: ${{ secrets.BORG_REPO }}
+      BORG_PASS: ${{ secrets.BORG_PASS }}
+      FTP_DIR: ostra
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install prerequisites and create initial configuration
+        run: |
+          sudo apt-get update
+          sudo apt-get install borgbackup
+          sudo pip install borgmatic
+          curl -O https://downloads.rclone.org/rclone-current-linux-amd64.zip
+          unzip rclone-current-linux-amd64.zip
+          sudo cp rclone-*-linux-amd64/rclone /usr/local/bin
+          sudo chmod +x /usr/local/bin/rclone
+          rclone config create --non-interactive gamecon ftp \
+            host=$RCLONE_HOST \
```

Comment:
Poprosím tě o malý návod. Tohle jsou přístupy k FTP? Mate mě ten prefix RCLONE_

Reply:
Ano, jsou to FTP přístupy, jen jsou ve formátu pro rclone, tj. v konfiguraci má heslo prohnané skrze `rclone obscure` (takže base64). Proto ten prefix.

Reply:
Poprosím Tě o readme s návodem, co která proměnná znamená a hlavně jak ji případně obnovit.

Reply:
Určitě, chystám se ještě sepsat i návod na obnovu ze zálohy. K dořešení také je, kam si bezpečně ty správné hodnoty odložit, přeci jen tam máme pár tajných hodnot.

Reply:
Návod přidán, snad se mi povedla nějaká srozumitelná verze.